packages/lib-classifier/src/store/feedback/strategies/single-answer-question/reducer.js

```diff
@@ -1,5 +1,5 @@
 function singleAnswerQuestionReducer (rule, answer) {
-  const answerId = (answer > -1) ? answer : -1
+  const answerId = (answer > -1) && (answer !== null) ? answer : -1
```

Comment:
I'm not sure if the answer could be `undefined`. 
Once we're content with this or a different way to update the reducer to handle `null` we can update PFE as well.